### PR TITLE
Added Rule VPC_ENDPOINT_MANUAL_ACCEPTANCE.

### PR DESCRIPTION
**Pull request by pronoyroy**
I confirm these files are made available under CC0 1.0 Universal (https://creativecommons.org/publicdomain/zero/1.0/legalcode)

*Issue #, if available:*

*Description of changes:*
Created the periodic Rule VPC_ENDPOINT_MANUAL_ACCEPTANCE and the test file.

### CODE REVIEW

**Comment by jongogogo** [comment]
Who is the BR?

**Comment by pronoyroy** [comment]
Hi, Rafih is the BR

**Comment by rafihm** [comment]
Thanks for the update @pronoyroy 
@jongogogo Please review the PR.

File: python/VPC_ENDPOINT_MANUAL_ACCEPTANCE/VPC_ENDPOINT_MANUAL_ACCEPTANCE.py [hunk 1]
```diff
@@ -0,0 +1,401 @@
...
+    for service in get_endpoint_services(ec2_client):
+        if not service['Owner'] == 'amazon':
+            if service['AcceptanceRequired']:
+                evaluations.append(build_evaluation(service['ServiceName'], 'COMPLIANT', event, DEFAULT_RESOURCE_TYPE))
+                continue
+            evaluations.append(build_evaluation(service['ServiceName'], 'NON_COMPLIANT', event, DEFAULT_RESOURCE_TYPE, annotation='The Endpoint Service has "AcceptanceRequired" set to False.'))
+    return evaluations
+
+def get_endpoint_services(ec2_client):
+    endpoint_services = []
```
Comment (jongogogo) [inline python/VPC_ENDPOINT_MANUAL_ACCEPTANCE/VPC_ENDPOINT_MANUAL_ACCEPTANCE.py]: Change the code to be closer to the below (uplifted from somewhere else)
```
    old_evals = []
    next_token = ''
    while True:
        compliance_details = config_client.get_compliance_details_by_config_rule(
            ConfigRuleName=event['configRuleName'],
            ComplianceTypes=['COMPLIANT', 'NON_COMPLIANT'],
            Limit=100,
            NextToken=next_token)

        old_evals.extend(compliance_details['EvaluationResults'])
        next_token = compliance_details.get('nextToken', '')
        if not next_token:
            break
```

File: python/VPC_ENDPOINT_MANUAL_ACCEPTANCE/VPC_ENDPOINT_MANUAL_ACCEPTANCE.py [hunk 2]
```diff
@@ -0,0 +1,401 @@
...
+    while next_token != '':
+        next_token = ''
+        endpoint_services.extend(response['ServiceDetails'])
+        if 'NextToken' in response:
+            next_token = response['NextToken']
+            response = ec2_client.describe_vpc_endpoint_services(NextToken=next_token)
+    return endpoint_services
+
+def evaluate_parameters(rule_parameters):
+    """
```
Comment (jongogogo) [inline python/VPC_ENDPOINT_MANUAL_ACCEPTANCE/VPC_ENDPOINT_MANUAL_ACCEPTANCE.py]: No need for comments

**Comment by jongogogo** [comment]
When is the next update?

**Comment by jongogogo** [comment]
Anyone?

**Comment by rafihm** [comment]
Hey @pronoyroy Can you please check the comments. 
Thanks

**Comment by pronoyroy** [comment]
Hi, really very sorry for the delay. Have made the changes. Can you please have a look. Will make any further changes if required at the very earliest.

**Pull request merged**